src/transaction/log_manager.c

```diff
@@ -10064,8 +10065,14 @@ class log_check_ha_delay_info_daemon_task : public cubthread::entry_task
 class log_flush_daemon_task : public cubthread::entry_task
 {
   public:
+    log_flush_daemon_task ()
+    {
+      m_p_log_flush_lsa = cubtx::single_node_group_complete_manager::get_instance ();
```

Comment:
What happens on master/slave nodes? Is single node gcm instantiated?

Reply:
Master/slave case will be considered with next patch - log_complete, when my intention is to activate also master/slave and to have minimal testing. 
For clarification, I will add TODO comment here.

Reply:
It would be more safe and clear to add something like:
```c
  if (HA_DISABLED ())
    {
      m_p_log_flush_lsa = cubtx::single_node_group_complete_manager::get_instance ();
    }
```
```
  if (m_p_log_flush_lsa != NULL)
    {
      m_p_log_flush_lsa->notify_log_flush_lsa (&nxio_lsa);
    }
```

Reply:
As I said, this kind of code will be added in next patch with testing.  Also, initialization will consider similar code.

Let's not forget that GC is disabled so far and we need to activate it for single SERVER_MODE/SA, testing first, then master/slave testing.